packages/web-components/src/components/va-promo-banner/va-promo-banner.tsx

```diff
@@ -0,0 +1,170 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  h,
+  Prop,
+  State,
+} from '@stencil/core';
+
+const DISMISSED_PROMO_BANNERS_LOCAL_STORAGE_KEY = 'DISMISSED_PROMO_BANNERS';
+@Component({
+  tag: 'va-promo-banner',
+  styleUrl: 'va-promo-banner.css',
+  shadow: true,
+})
+export class VaPromoBanner {
+  @Element() el: HTMLElement;
+
+  /**
+   * The rendering custom markup via the slot instead of the `<a>` with `text` in it
+   * */
+  @Prop() renderCustom: boolean = false;
```

Comment:
Could we just determine this automatically based on the slotted content? We may be able to see if it is an actual `Node` or just text content and then render with the `<a>` wrapping the `<slot>` or not.

Reply:
Removed per: https://github.com/department-of-veterans-affairs/component-library/pull/297#issuecomment-1051216595